### PR TITLE
[11.0][IMP] account_bank_statement_import_paypal

### PR DESCRIPTION
**Pull request by alexey-pelykh**
Backport of enhancements from v12, depends on https://github.com/OCA/web/pull/1572

### CODE REVIEW

**Comment by pedrobaeza** [comment]
We are not interested in this version. Our customers with this module are already in v12.

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+web https://github.com/brainbeanapps/web 11.0-mig-web_widget_dropdown_dynamic
+server-ux
+connector-jira
```
Comment (gurneyalex) [inline oca_dependencies.txt:2]: why are you adding connector-jira here?

Comment (alexey-pelykh) [reply oca_dependencies.txt:2]: Unfortunately, `multi_step_wizard` lives there in v11 :(

File: account_bank_statement_import_paypal/__manifest__.py [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
+    'website': 'https://github.com/OCA/bank-statement-import',
+    'author':
+        'Akretion, '
+        'Brainbean Apps, '
+        'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'depends': [
+        'account_bank_statement_import',
+        'multi_step_wizard',
```
Comment (gurneyalex) [inline account_bank_statement_import_paypal/__manifest__.py:20]: I don't see this module in https://github.com/OCA/server-ux/tree/11.0 (its in 12.0 and there's a PR to migrate it to 13.0 but no backport to 11.0 AFAICT)

Comment (alexey-pelykh) [reply account_bank_statement_import_paypal/__manifest__.py:20]: It's there originally https://github.com/OCA/connector-jira/tree/11.0

**Review by ghost — APPROVED** [review]

**Pull request merged**